src/OnboardingSPA/components/Drawer/DrawerPanel/NavPrimary.js

```diff
@@ -27,6 +28,9 @@ const NavPrimary = () => {
 			<div className="nfd-onboarding-drawer__panel-menu">
 				<ul className="nfd-onboarding-drawer__panel-routes">
 					{ topSteps.map( ( step ) => {
+						if ( step === stepTheFork ) {
+							return false;
+						}
```

Comment:
We should define a key in the `step.js` for the fork step, such as `drawerNavigation: false;`, and then implement the necessary checks wherever needed.

Reply:
Sure